dracut/30ignition/ignition-setup.sh

```diff
@@ -1,22 +1,27 @@
 #!/bin/bash
+set -eux
 
-set -e
+copy_file() {
+    src=$1; dst=$2
+    if [ -f $src ]; then
```

Comment:
`$src`, `$1`, and `$2` should be quoted for future-proofing.

Reply:
fixed in next upload